docker/tests/Dockerfile

```diff
@@ -25,12 +20,20 @@ RUN chmod +x /start.sh
 COPY ./docker/tests/check-code.sh /check-code.sh
 RUN chmod +x /check-code.sh
 
-COPY ./ /app
-
 COPY ./envs/web.template ./envs/web
 
+COPY poetry.lock pyproject.toml /app/
+
 WORKDIR /app/
 
+# not recommended in docs to install poetry globally but as this is a VM this is fine

```

Comment:
This comment doesn't look right - GBIS doesn't run in a VM

Reply:
Yes, sorry, I think this was an artefact from the framework we were using for this ticket. I'll remove these

Reply:
I do think the comments might still be warranted if you can reword